### PR TITLE
Documentation for APIv4: GET /teams/name/{name}

### PR DESCRIPTION
**Pull request by saturninoabril**
Documentation for APIv4: GET /teams/name/{name}

Mattermost/platform [Github issue: #5414](https://github.com/mattermost/platform/issues/5414)

### CODE REVIEW

File: source/v4/teams.yaml [hunk 1]
```diff
@@ -69,6 +69,35 @@
         '404':
           $ref: '#/responses/NotFound'
 
+  '/teams/name/{name}':
+    get:
+      tags:
+        - teams
+      summary: Get a team based on name string query parameter
```
Comment (jwilander) [inline source/v4/teams.yaml]: Can we shorten this to `Get team by name`? This will end up in a sidebar so it's nice to keep it short

File: source/v4/teams.yaml [hunk 2]
```diff
@@ -96,6 +125,45 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+  '/teams/{team_id}/members':
+    get:
+      tags:
+        - teams
+      summary: Get a page of a list of team members based on query string parameters - team id, page and per page.
```
Comment (jwilander) [inline source/v4/teams.yaml]: Same here, can we make this `Get team members` and then move the extra explanation into the description below?

Comment (saturninoabril) [inline source/v4/teams.yaml]: Noted. See changes made. thanks!

**Comment by saturninoabril** [comment]
I rebased and fixed commits.

See newly opened commit: [Issue #105 ](https://github.com/mattermost/mattermost-api-reference/pull/105)
